### ISSUE TITLE
Error adding tracking code on order

### ISSUE THREAD

**Issue by yopop666** [Sylius/Sylius#4552]
In Sylius-standard version 17
Catchable Fatal Error: Argument 1 passed to Sylius\Bundle\ResourceBundle\Controller\ResourceController::findOr404() must be an instance of Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration, instance of Symfony\Component\HttpFoundation\Request given, called in vendor/sylius/sylius/src/Sylius/Bundle/ShippingBundle/Controller/ShipmentController.php on line 22 and defined

I fixed this issue with code
```<?php

/*
 * This file is part of the Sylius package.
 *
 * (c) Paweł Jędrzejewski
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

namespace Sylius\Bundle\ShippingBundle\Controller;

use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
use Sylius\Component\Shipping\ShipmentTransitions;
use Symfony\Component\HttpFoundation\Request;

use FOS\RestBundle\View\View;

class ShipmentController extends ResourceController
{
    public function shipAction(Request $request)
    {
        $configuration = $this->requestConfigurationFactory->create($this->metadata, $request);
        $shipment = $this->findOr404($configuration);
        $form = $this->createForm('sylius_shipment_tracking', $shipment);

        if ($form->submit($request)->isValid()) {
            $this
                ->get('sm.factory')
                ->get($shipment, ShipmentTransitions::GRAPH)
                ->apply(ShipmentTransitions::SYLIUS_SHIP)
            ;
            $shipment = $form->getData();

            $this->manager->flush();

            $this->addFlash('success', 'sylius.shipment.ship.success');

            return $this->redirectHandler->redirectToResource($configuration,$shipment);
        }

        $view = View::create()
            ->setData([
                'metadata' => $this->metadata,
                'resource' => $resource,
                $this->metadata->getName() => $resource,
                'form' => $form->createView(),

                'shipment' => $shipment,
                'shipment_tracking_form' => $form->createView(),
            ])
            ->setTemplate($configuration->getTemplate('show.html'))
        ;

        return $this->viewHandler->handle($configuration, $view);
    }
}
```

**Comment by lighta971** [comment]
Already fixed in #4530

**Comment by pjedrzejewski** [comment]
Thanks for reporting Andrei!
